docs/svelte.config.js

```diff
@@ -39,7 +39,6 @@ const config = {
             '@web3-react/metamask',
             '@web3-react/network',
             '@web3-react/walletconnect',
-            '@web3-react/types',
```

Comment:
@leightkt have you confirmed all other wallets thats use `@web3-react/types` still work within the docs?
I dont know which wallets this was added here for but was added to this list for some wallet

Reply:
@Adamj1232 Looking in the yarn.lock file- it looks like only @infinitywallet/infinity-connector and @web3-react/abstract-connector (an @infinitywallet/infinity-connector) use @web3-react/types

Reply:
Hmm strange as that was added prior to the infinity wallet 🤔 may be an outdated dep
Were you able to test walletConnect by chance? I see they are a web3-react package also.
Can we remove any of the other web3-react deps in there?